### PR TITLE
add droneci docs

### PR DESCRIPTION
**Pull request by joshmarinacci**

### CODE REVIEW

**Comment by gewenyu99** [comment]
Screenshot 2024-09-16 at 1 58 47 PM
Screenshot 2024-09-16 at 1 58 47 PM
Remember to add something here

**Comment by gewenyu99** [comment]
Screenshot 2024-09-16 at 1 59 30 PM
Screenshot 2024-09-16 at 1 59 30 PM
Also remember to fix supademo after merging

File: flaky-tests/ci-providers/droneci.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+---
+
+# Drone CI
+
+## Getting Started
+
+You can use the Flaky Tests CLI within your [Drone CI](https://www.drone.io/) pipelines to upload and analyze your test results.
+
+{% hint style="info" %}
+The Trunk Flaky Tests CLI currently only supports x86_64 and arm64 for both Linux and macOS. If you have another use case, please get in touch with support at [https://slack.trunk.io](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation does not use cached test results and does not automatically retry failing tests.
```
Comment (gewenyu99) [inline flaky-tests/ci-providers/droneci.md:12]: This should be a hyperlink like
```suggestion
The Trunk Flaky Tests CLI currently only supports x86_64 and arm64 for both Linux and macOS. If you have another use case, please get in touch with support in our [community Slack](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation does not use cached test results and does not automatically retry failing tests.
```
```suggestion
The Trunk Flaky Tests CLI currently only supports x86_64 and arm64 for both Linux and macOS. If you have another use case, please get in touch with support at [https://slack.trunk.io](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation does not use cached test results and does not automatically retry failing tests.
```

File: flaky-tests/ci-providers/droneci.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+{% endhint %}
+
+### Create a Drone CI Pipeline
+
+Create a Drone CI pipeline (or modify an existing one) to run the tests that you want to monitor. The pipeline should produce a test report in [**JUnit XML**](https://github.com/testmoapp/junitxml) format. Most testing frameworks support XML output. See [Testing Framework Configuration](../frameworks/) for guides for common testing frameworks. Make sure that your test invocation doesn't use cached test results, and doesn't automatically retry failing tests.
```
Comment (gewenyu99) [inline flaky-tests/ci-providers/droneci.md:17]: Does the `../frameworks/` relative link work?

Comment (gewenyu99) [reply flaky-tests/ci-providers/droneci.md:17]: (doesn't in the GitBooks preview

File: flaky-tests/ci-providers/droneci.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+```
+{% endcode %}
+{% endtab %}
+
+{% endtabs %}
+
+
+
+In the config about we have added two additional commands to download the latest release of the `trunk-analytics-cli` and
+then run it to upload the test output XML file. The TRUNK\_ORG\_SLUG and TRUNK\_API\_TOKEN variables are filled in at runtime by
```
Comment (gewenyu99) [inline flaky-tests/ci-providers/droneci.md:143]:
```suggestion
then run it to upload the test output XML file. The `TRUNK\_ORG\_SLUG` and `TRUNK\_API\_TOKEN` variables are filled in at runtime by
```
```suggestion
then run it to upload the test output XML file. The TRUNK\_ORG\_SLUG and TRUNK\_API\_TOKEN variables are filled in at runtime by
```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: flaky-tests/ci-providers/droneci.md (hunk 2 above)
Comment (joshmarinacci) [reply flaky-tests/ci-providers/droneci.md]: I fixed it.

**Review by gewenyu99 — APPROVED** [review]

**Pull request merged**